pkgs/servers/nextcloud/packages/nextcloud-apps.json

```
@@ -37,4 +37,5 @@
 , "unsplash": "agpl3Only"
 , "user_oidc": "agpl3Plus"
 , "user_saml": "agpl3Plus"
+, "whiteboard": "agpl3Only"
```

```suggestion
, "whiteboard": "agpl3Plus"
```
https://github.com/nextcloud/whiteboard/blob/main/package.json#L4